serialization/include/rp_ser.h

```diff
@@ -177,10 +171,9 @@ struct rp_ser {
  * @param[in] _name Instance name.
  * @param[in] _lock_type Os lock data type. For example in Zephyr struct k_sem can be used.
  * @param[in] _endpoint_num Endpoint number, used for transport endpoint identify.
- * @param[in] _endpoint_stack_size Endpoint thread stack size.
- * @param[in] _endpoint_thread_prio Endpoint thread priority.
+ * @param[in] __VA_ARGS__ Transport specific parameters.
  */
-#define RP_SER_DEFINE(_name, _lock_type, _endpoint_num, _endpoint_stack_size, _endpoint_thread_prio) \
+#define RP_SER_DEFINE(_name, _lock_type, _endpoint_num, ...) \
```

Comment:
Do we really need this `...` argument. It migth be a little confusing for user ?

Reply:
Those are platform specific parameters (or maybe transport specific), e.g. baremetal application will not use it. I used `...` to allow passing any parameters that may vary between different transport implementations.

So far I see two options:
* stack size and priority for RTOS, e.g. Zephyr or FreeRTOS,
* no parameters for baremetal

I can switch back to `_stack_size` and `_prio` instead of `...` if you think that it will be more clear to user. `_stack_size` and `_prio` will be ignored in baremetal. If some other options will show up then we can adjust the parameters later.

Reply:
Done